/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/urllib3/__init__.py:35: NotOpenSSLWarning: urllib3 v2 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020
  warnings.warn(
[2024-07-25 12:38:13,186] MBP-de-sidi/INFO/locust.main: Starting web interface at http://0.0.0.0:8089
[2024-07-25 12:38:13,191] MBP-de-sidi/INFO/locust.main: Starting Locust 2.29.1
Traceback (most recent call last):
  File "src/gevent/greenlet.py", line 908, in gevent._gevent_cgreenlet.Greenlet.run
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/locu[2024-07-25 13:10:43,090] MBP-de-sidi/INFO/locust.runners: Ramping to 1 users at a rate of 1.00 per second
[2024-07-25 13:10:43,090] MBP-de-sidi/INFO/locust.runners: All users spawned: {"LoadTestUser": 1} (1 total users)
 self.start()
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/gevent/baseserver.py", line 336, in start
    self.init_socket()
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/gevent/pywsgi.py", line 1668, in init_socket
    StreamServer.init_socket(self)
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/gevent/server.py", line 173, in init_socket
    self.socket = self.get_listener(self.address, self.backlog, self.family)
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/gevent/server.py", line 185, in get_listener
    return _tcp_listener(address, backlog=backlog, reuse_addr=cls.reuse_addr, family=family)
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/gevent/server.py", line 264, in _tcp_listener
    sock.bind(address)
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/gevent/_socketcommon.py", line 547, in bind
    return self._sock.bind(address)
OSError: [Errno 48] Address already in use: ('', 8089)
2024-07-25T10:38:13Z <Greenlet at 0x106c4fd60: <bound method WebUI.start_server of <locust.web.WebUI object at 0x106c56fa0>>> failed with OSError

[2024-07-25 12:38:13,195] MBP-de-sidi/CRITICAL/locust.web: Unhandled exception in greenlet: <Greenlet at 0x106c4fd60: <bound method WebUI.start_server of <locust.web.WebUI object at 0x106c56fa0>>>
Traceback (most recent call last):
  File "src/gevent/greenlet.py", line 908, in gevent._gevent_cgreenlet.Greenlet.run
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/locust/web.py", line 538, in start_server
    self.server.serve_forever()
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/gevent/baseserver.py", line 398, in serve_forever
    self.start()
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/gevent/baseserver.py", line 336, in start
    self.init_socket()
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/gevent/pywsgi.py", line 1668, in init_socket
    StreamServer.init_socket(self)
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/gevent/server.py", line 173, in init_socket
    self.socket = self.get_listener(self.address, self.backlog, self.family)
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/gevent/server.py", line 185, in get_listener
    return _tcp_listener(address, backlog=backlog, reuse_addr=cls.reuse_addr, family=family)
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/gevent/server.py", line 264, in _tcp_listener
    sock.bind(address)
  File "/Users/sidimohammedkaddour/PycharmProjects/Contract Net/.venv/lib/python3.9/site-packages/gevent/_socketcommon.py", line 547, in bind
    return self._sock.bind(address)
OSError: [Errno 48] Address already in use: ('', 8089)
[2024-07-25 12:38:13,196] MBP-de-sidi/INFO/locust.main: Shutting down (exit code 2)
Type     Name                                                                          # reqs      # fails |    Avg     Min     Max    Med |   req/s  failures/s
--------|----------------------------------------------------------------------------|-------|-------------|-------|-------|-------|-------|--------|-----------
--------|----------------------------------------------------------------------------|-------|-------------|-------|-------|-------|-------|--------|-----------
         Aggregated                                                                         0     0(0.00%) |      0       0       0      0 |    0.00        0.00

Response time percentiles (approximated)
Type     Name                                                                                  50%    66%    75%    80%    90%    95%    98%    99%  99.9% 99.99%   100% # reqs
--------|--------------------------------------------------------------------------------|--------|------|------|------|------|------|------|------|------|------|------|------
--------|--------------------------------------------------------------------------------|--------|------|------|------|------|------|------|------|------|------|------|------

